pkg/grpc/authn/tokenbased/extractor.go

```diff
@@ -37,7 +37,7 @@ func (e *extractor) IdentityForRequest(ctx context.Context, ri requestinfo.Reque
 	}
 	token, err := e.validator.Validate(ctx, rawToken)
 	if err != nil {
-		logging.GetRateLimitedLogger().WarnL(
+		logging.GetRateLimitedLogger().DebugL(
```

Comment:
While I'm not sure why these are needed, there are other extractors we should do this to if we keep the logging: servicecerttoken, basic